src/components/common/header/Notification/NotificationItem/NotificationItem.tsx

```
@@ -3,32 +3,26 @@ import {
-    type: 'command' | 'pass/nonPass' | 'inquiry';
-    id: number;
-    time: string;
-    message: string;
-    nickName?: string;
-    pass?: boolean;
-  };
+  item: AlarmAll;
 }
 
-const NotificationItem = ({ NotificationData }: NotificationItemProps) => {
+const NotificationItem = ({ item }: NotificationItemProps) => {
```

ESLint

[error] 'item' is defined but never used.

(@typescript-eslint/no-unused-vars)